pytket/extensions/iqm/backends/iqm.py

```diff
@@ -339,10 +333,13 @@ def get_metadata(self, handle: ResultHandle, **kwargs: KwargTypes) -> Metadata:
 
 
 def _as_node(qname: str) -> Node:
-    assert qname.startswith("QB")
-    x = int(qname[2:])
-    assert x >= 1
-    return Node(x - 1)
+    if qname == "COMP_R":
```

Comment:
We currently fail with devices (such as "deneb") that contain "resonator" qubits. I will make a new issue for this.